client/web/src/insights/components/form/repositories-field/components/flex-textarea/FlexTextArea.tsx

```diff
@@ -0,0 +1,57 @@
+import classNames from 'classnames'
+import React, { forwardRef, InputHTMLAttributes, Ref, useEffect, useImperativeHandle, useRef, useState } from 'react'
+
+import styles from './FlexTextarea.module.scss'
+
+export type IProps = {
+    initialRow?: number
+    minRows?: number
+    maxRows?: number
+} & InputHTMLAttributes<HTMLInputElement>
+
+/**
+ * Flexible and auto-growable textarea element.
+ */
+export const FlexTextarea = forwardRef((props: IProps, reference: Ref<HTMLInputElement | null>) => {
```

Comment:
```suggestion
export const FlexTextArea = forwardRef((props: IProps, reference: Ref<HTMLInputElement | null>) => {
```

To conform with e.g. `HTMLTextAreaElement`